assets/stylesheets/components/_team.scss

```diff
@@ -0,0 +1,44 @@
+.members {
+	.member {
+		
+	}
+}
+.team {
+	article.member {
```

Comment:
@nonlinear to avoid specificity issues, it is generally advisable to just use the class name. no need to list the element.